python/shotgun_model/shotgun_hierarchy_item.py

```diff
@@ -0,0 +1,108 @@
+# Copyright (c) 2016 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+import sgtk
+
+from .shotgun_standard_item import ShotgunStandardItem
+
+
+class ShotgunHierarchyItem(ShotgunStandardItem):
+    """
+    A subclass of ``ShotgunStandardItem`` with access to data provided via
```

Comment:
I've added a warning here about creating manually, similar to the warning in `ShotgunStandardItem`. That sufficient?